src/Sulu/Bundle/PageBundle/Admin/PageAdmin.php

```diff
@@ -316,7 +316,7 @@ public function getSecurityContexts()
             }
 
             $system = $security->getSystem();
-            if (!$system) {
+            if (!$webspace->hasWebsiteSecurity()) {
```

Comment:
I don't think this is the correct place to handle this. I think the system if configured should still exist to create roles but the section not be shown on the permission tab. The problem with this solution is also if I have a webspace A with:

```xml
    <security permission-check="true">
        <system>Website</system>
    </security>
```

and a webspace B with:

```xml
    <security>
        <system>Website</system>
    </security>
```

both webspace would show the `Website` Permission Tab. So I think we need to handle this in the JS component, where we currently handles that only the specific system is shown, we need to check for `hasWebsiteSecurity`, don't see currently another solution to handle this.

Reply:
@alexander-schranz Wanted to do it like that at first, but somehow was not able to, the break seems to had a good influence :see_no_evil: 

However, now I've implemented it that way, and not really happy either... It feels like our config request returns something wrong (because it looks like you can set these permissions on these systems) and you have to fix that in the frontend by combining it with some other information. From that point of view I liked the other solution better, although what you described might be a problem (although I am quite sure that the setup you've described there would break at other points as well).